src/memray/commands/run.py

```diff
@@ -128,8 +132,12 @@ def _run_with_socket_output(args: argparse.Namespace) -> None:
 
 def _run_with_file_output(args: argparse.Namespace) -> None:
     if args.output is None:
-        output = f"memray-{os.path.basename(args.script)}.{os.getpid()}.bin"
-        filename = os.path.join(os.path.dirname(args.script), output)
+        script_name = args.script
+        if args.run_as_cmd:
+            script_name = "cmd_script"
```

Comment:
The interpreter calls a command given with `-c` "<string>" - you can see it in tracebacks like:
```
~>python -c "1/0"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ZeroDivisionError: division by zero
```

I don't think I want to include the `<` and `>` in the name here, since those are shell metacharacters that can cause havoc if people forget to quote them, but I think I'd prefer calling this "string" instead of "cmd_script", to at least somewhat match what the interpreter is doing.
```suggestion
            script_name = "string"
```

Reply:
could be `string`, `command`, `cmd`, not an issue